management-service/src/test/groovy/pl/pwr/zpi/notifications/discord/DiscordReceiverServiceTest.groovy

```diff
@@ -0,0 +1,157 @@
+package pl.pwr.zpi.notifications.discord
+
+import pl.pwr.zpi.notifications.common.ConfidentialTextEncoder
+import pl.pwr.zpi.notifications.discord.dto.DiscordReceiverDTO
+import pl.pwr.zpi.notifications.discord.dto.UpdateDiscordReceiverRequest
+import pl.pwr.zpi.notifications.discord.entity.DiscordReceiver
+import pl.pwr.zpi.notifications.discord.repository.DiscordRepository
+import pl.pwr.zpi.notifications.discord.service.DiscordReceiverService
+import spock.lang.Specification
+
+class DiscordReceiverServiceTest extends Specification {
+
+    DiscordRepository discordRepository
+    ConfidentialTextEncoder confidentialTextEncoder
+    DiscordReceiverService discordReceiverService
+
+    def setup() {
+        confidentialTextEncoder = Mock()
+        discordRepository = Mock()
+        discordReceiverService = new DiscordReceiverService(discordRepository, confidentialTextEncoder)
+        discordReceiverService.WEBHOOK_URL_REGEX = "https://discord.com/api/webhooks/[0-9]+/[a-zA-Z0-9_-]+"
```

Comment:
what if the regex changes in the implementation?

Reply:
I will pull that from properties

Reply:
You need to update it here too